examples/mnist/train.py

```diff
@@ -51,38 +51,24 @@ def __call__(self, x):
     return x
 
 
-def compute_metrics(logits, labels):
-  loss = jnp.mean(
-      optax.softmax_cross_entropy(
-          logits=logits, labels=jax.nn.one_hot(labels, 10)))
-  accuracy = jnp.mean(jnp.argmax(logits, -1) == labels)
-  metrics = {
-      'loss': loss,
-      'accuracy': accuracy,
-  }
-  return metrics
-
-
 @jax.jit
-def train_step(state, batch):
-  """Train for a single step."""
+def apply_model(state, images, labels):
+  """Compute gradientsh loss and accuracy for a single batch."""
```

Comment:
`"""Computes gradients, loss, accuracy and for a single batch."""`

Reply:
done